tests/test_run_sign.py

```diff
@@ -32,7 +32,7 @@ def test_usage(self, *mocks):
 
     @patch("os.getcwd", return_value="curdir")
     @patch("argparse._sys.argv", ["run_sign.py", BUILD_MANIFEST])
-    @patch("run_sign.Signer", return_value=MagicMock())
+    @patch("sign_workflow.sign_artifacts.SignArtifacts", return_value=MagicMock())
```

Comment:
How did you figure out the trick to get this working?

Reply:
It's not working yet.

Reply:
It's fixed now